django/thunderstore/community/templates/community/packagelisting_detail.html

```diff
@@ -65,14 +66,18 @@
     </div>
 {% endif %}
 
-{% if object.is_rejected %}
+{% endcache %}
+
+{% if show_review_status and object.is_rejected %}
 <div class="card text-white bg-danger mt-2">
     <div class="card-body">
         <h4 class="card-title">
             Package rejected
         </h4>
         <p class="card-text">
-            This package has been rejected by site or community moderators
+            This package has been rejected by site or community moderators.
+            If you think this is a mistake, please reach out to the moderators in
+            <a href="https://discord.thunderstore.io/">our Discord server</a>
```

Comment:
Should we have a dedicated channel and/or more precise instructions for this? I'm afraid the way it's worded now will get us questions related to moderation on unrelated channels.